src/pages/instances/InstanceSnapshots.tsx

```diff
@@ -54,7 +83,9 @@ const InstanceSnapshots: FC<Props> = ({ instance }) => {
 
   const { project, isLoading } = useProject();
 
-  const snapshotsDisabled = project?.config["restricted.snapshots"] === "block";
+  const snapshotsDisabled =
+    project?.config["restricted.snapshots"] === "block" ||
+    project?.config["restricted"] === "true"; // Saw this on the demo server restricting snapshot creation;
```

Comment:
If the project is restricted, we have to check if restricted.snapshots is set to block or empty, because empty equals to block. We should not rely on the restricted setting alone, it doesn't say if snapshots are allowed or not.

Reply:
created a function for it in utils/snapshots.tsx